src/components/UserProfile.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Button,
+  Card,
+  CardContent,
+  CardActions,
+  Table,
+  TableRow,
+  TableCell,
+  getImageListItemBarUtilityClass,
+} from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+// import { getUserProfile } from './api';
+import { Avatar } from '@mui/material';
+import axios from 'axios';
+
+export default function PersonalProfile() {
+  const navigate = useNavigate();
+  const { user, status } = useAuth();
+  const [userData, setUserData] = useState(null); 
+  const [showCreditCardInfo, setShowCreditCardInfo] = useState(false);
+
+  
+  const handleSeeMoreClick = () => {
+    setShowCreditCardInfo(!showCreditCardInfo);
+  };
+
+  useEffect(() => {
+    if (status === AuthStatus.LoggedIn) {
+
+      axios.get('http://localhost:8000/api/v1/user') 
+      .then((response) => {
+          setUserData(response.data); // Update user data in state
+        })
+        .catch((error) => {
+          // Handle errors, e.g., display an error message
+          console.error('Error fetching user data:', error);
+        });
+    }
+  }, 
+  [status]);
+
+
+  return (
+    <Card className="mt-2 border-0 rounded-0 shadow-sm">
+      <CardContent>
+        <h3 className="text-uppercase">My Profile</h3>
+        <div className="text-center">
+          <Avatar
+            src={
+              userData?.profileImage?.url ||
+              'https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-chat/ava1-bg.webp'
+            }
+            alt="user profile"
+            className="img-fluid rounded-circle"
+            sx={{
+              width: '200px',
+              height: '200px',
+              maxWidth: '200px',
+              maxHeight: '200px',
+            }}
+          />
+        </div>
+        <Table responsive striped hover className="text-center mt-5">
```

Comment:
There are errors in the dev console about the `responsive`, `striped`, and `hover` attributes that you have here. 

I think you're thinking of bootstrap where those attributes are available. But in Material-UI they are not.

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/4f073f99-fa8c-4f4d-98a7-5054cc3c14b8)

```suggestion
        <Table className="text-center mt-5">
```